### PR TITLE
codesmells fixes

### PR DESCRIPTION
**Pull request by kapoliub**

### CODE REVIEW

File: __tests__/user/user.mutation.test.js [hunk 1]
```diff
@@ -439,7 +438,6 @@ describe('User`s mutation restictions tests', () => {
     lastName = 'Markelo';
     email = '1xamp31d2v1@gmail.com';
     password = 'qwertY123';
```
Comment (nadiyafomenko) [inline __tests__/user/user.mutation.test.js:440]: please move constants to the variables file

Comment (kapoliub) [reply __tests__/user/user.mutation.test.js]: fixed

**Review by koropalov — APPROVED** [review]

File: __tests__/news/news.mutation.test.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ describe('News mutations tests', () => {
           `,
           variables: { news },
         })
-        .then(res => res)
+        .then(response => response)
```
Comment (maksym-strus) [inline __tests__/news/news.mutation.test.js:65]: Make sense to delete this **then**

File: __tests__/news/news.mutation.test.js [hunk 3]
```diff
@@ -123,7 +123,7 @@ describe('News mutations tests', () => {
 
           variables: { news },
         })
-        .then(res => res)
+        .then(response => response)
```
Comment (maksym-strus) [inline __tests__/news/news.mutation.test.js:126]: Make sense to delete this then

File: __tests__/news/news.mutation.test.js [hunk 4]
```diff
@@ -209,7 +209,7 @@ describe('News mutations tests', () => {
             `,
             variables: { id: newsDoesNotExistId, news: newsUpdateData },
           })
-          .then(res => res)
+          .then(response => response)
```
Comment (maksym-strus) [inline __tests__/news/news.mutation.test.js:212]: Make sense to delete this then

File: __tests__/news/news.mutation.test.js [hunk 5]
```diff
@@ -247,7 +247,7 @@ describe('News mutations tests', () => {
             `,
             variables: { id: wrongId, news: existingNews },
           })
-          .then(res => res)
+          .then(response => response)
```
Comment (maksym-strus) [inline __tests__/news/news.mutation.test.js:250]: Make sense to delete this then

File: __tests__/news/news.mutation.test.js (hunk 2 above)
Comment (kapoliub) [reply __tests__/news/news.mutation.test.js]: fixed

File: __tests__/news/news.mutation.test.js (hunk 3 above)
Comment (kapoliub) [reply __tests__/news/news.mutation.test.js]: fixed

File: __tests__/news/news.mutation.test.js (hunk 4 above)
Comment (kapoliub) [reply __tests__/news/news.mutation.test.js]: fixed

File: __tests__/news/news.mutation.test.js (hunk 5 above)
Comment (kapoliub) [reply __tests__/news/news.mutation.test.js]: fixed

**Review by nadiyafomenko — APPROVED** [review]

**Review by nadiyafomenko — COMMENTED** [review]
Sonar fails, maybe it will be good to fix

**Review by maksym-strus — APPROVED** [review]

**Pull request merged**